CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Cromwell Change Log
 
+## 40_hotfix Release Notes
```

Comment:
Not totally sure what the release WDL would make of this but there doesn't seem to be a precedent. There are precedents for something like `40.1` however. 🙂